.changeset/spotty-olives-share.md

```diff
@@ -0,0 +1,7 @@
+---
+'@techdocs/cli': minor
+'@backstage/plugin-techdocs-node': minor
```

Comment:
could probably be a patch as well - if TechDocs folks are fine with `minor` it can stay `minor`.

Reply:
thanks for the review! 

I figured minor suits it better since it does add functionality by adding a new optional parameter instead of fixing/improving an existing piece, but semver is always open to interpretation anyway imo.

Reply:
thanks for the review! 

I figured minor suits it better since it does add functionality by adding a new optional parameter instead of fixing/improving an existing piece, but semver is always open to interpretation anyway imo.